conda/cli/common.py

```diff
@@ -77,6 +77,14 @@ def ensure_name_or_prefix(args, command):
         raise CondaValueError('either -n NAME or -p PREFIX option required,\n'
                               'try "conda %s -h" for more details' % command)
 
+def is_active_prefix(prefix):
+    """
+    Determines whether the args we pass in are pointing to the active prefix.
+    Can be used a validation step to make sure operations are not being
+    performed on the active prefix.
+    """
+    return prefix == context.active_prefix or prefix == env_name(context.active_prefix)
```

Comment:
Might be useful to apply `conda.common.path.paths_equal` here, to ensure cases like:

* Paths with relative markers (`..`) which would be equivalent
* Symlinked locations (some people do that... Y_Y)